src/Common/Commands.Common/Properties/Resources.Designer.cs

```diff
@@ -1207,11 +1208,11 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Internal Server Error.
+        ///   Looks up a localized string similar to public Server Error.
         /// </summary>
-        public static string InternalServerErrorMessage {
+        public static string publicServerErrorMessage {
```

Comment:
Similar to the previous comment.  Is the name change of this method intentional?